tasks/main.yml

```diff
@@ -14,6 +14,26 @@
     state: started
     enabled: yes
 
+- name: Copy Kibana SSL key
+  become: yes
+  copy:
+    content: "{{ kibana_server_ssl_key }}"
+    dest: "/etc/kibana/kibana_server_ssl.key"
+    owner: kibana
+    group: kibana
+    mode: 0600
+  notify: restart kibana
```

Comment:
Requested changes have been made to work when the SSL-related variables are not defined.

Personally, my needs are met by keeping it simple and defining the certificate in vars/main.yml and the key in vars/vault.yml.